<ui version="4.0" >
 <class>DeviceManager</class>
 <widget class="QMainWindow" name="DeviceManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bluetooth Device Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>687</width>
     <height>277</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="deviceListWidget" />
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Address:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="addressLabel" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="typeLabel" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Trust:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="trustLabel" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="statusLabel" >
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>55</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="iconLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_6" >
      <item>
       <widget class="QPushButton" name="newDevButton" >
        <property name="text" >
         <string>New...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton" >
        <property name="text" >
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>258</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Tasks</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="separator" />
    <addaction name="actionRemove" />
    <addaction name="actionConnect" />
    <addaction name="actionRemove_Trust" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuEdit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>305</y>
     <width>687</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionNew" >
   <property name="text" >
    <string>New...</string>
   </property>
  </action>
  <action name="actionRemove" >
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
  <action name="actionConnect" >
   <property name="text" >
    <string>Connect</string>
   </property>
  </action>
  <action name="actionRemove_Trust" >
   <property name="text" >
    <string>Remove Trust</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPair" >
   <property name="text" >
    <string>Pair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DeviceManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>659</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>516</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>DeviceManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>319</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
